packages/q/qt6base/xmake.lua

```diff
@@ -0,0 +1,13 @@
+package("qt6base")
+    set_kind("phony")
+    set_base("qtbase")
+
+    add_versions("6.3.0", "dummy")
+
+    on_load(function (package)
+        package:set("kind", "phony")
```

Comment:
Why set it twice?

Reply:
I can't recall, we can probably remove it